build.sbt

```diff
@@ -15,7 +15,7 @@ lazy val core = project
 
 lazy val docs = project.in(file("site")).enablePlugins(Http4sOrgSitePlugin)
 
-ThisBuild / tlBaseVersion := "0.10"
+ThisBuild / tlBaseVersion := "0.11"
```

Comment:
I wonder whether this could be a lint of some sort: if there's a newer tag on the current branch.

Reply:
Like a self-fixing lint? Or just a lint warning, that's what it currently does.
```
[info] looking for workflow definition in /home/runner/work/sbt-http4s-org/sbt-http4s-org/.github/workflows
java.lang.RuntimeException: Your tlBaseVersion 0.10 is behind the latest tag 0.11.0
```
https://github.com/http4s/sbt-http4s-org/runs/4952042260?check_suite_focus=true#step:7:23